modules/acquia_cms_headless/src/Plugin/AcquiaCmsHeadless/HeadlessApiKeys.php

```diff
@@ -128,7 +128,7 @@ public function getEntityData() {
     $query->tableSort($header);
     $query->pager(10);
 
-    return $query->execute();
+    return $query->accessCheck(FALSE)->execute();
```

Comment:
@deepakmishra2 can we update with **accessCheck(TRUE)** ??

Reply:
done